libs/core/langchain_core/messages/base.py

```diff
@@ -92,6 +92,28 @@ def get_lc_namespace(cls) -> list[str]:
         """
         return ["langchain", "schema", "messages"]
 
+    @property
+    def text(self) -> str:
+        """Get the text content of the message.
+
+        Returns:
+            The text content of the message.
+        """
+        if isinstance(self.content, str):
+            return self.content
+
+        # must be a list
+        blocks = [
+            block
+            for block in self.content
+            if isinstance(block, str)
+            or block["type"] == "text"
```

Comment:
Should we be safer about this or are we actually consistent enough to know it will have "type"?

Reply:
made safer ty